python/cuspatial/cuspatial/core/gis.py

```diff
@@ -84,19 +92,23 @@ def haversine_distance(p1_lon, p1_lat, p2_lon, p2_lat):
     """ Compute the haversine distances between an arbitrary list of lon/lat
     pairs
 
-    params
-    p1_lon: longitude of first set of coords
-    p1_lat: latitude of first set of coords
-    p2_lon: longitude of second set of coords
-    p2_lat: latitude of second set of coords
-
     Parameters
     ----------
-    {params}
+    p1_lon
+        longitude of first set of coords
+    p1_lat
+        latitude of first set of coords
+    p2_lon
+        longitude of second set of coords
+    p2_lat
+        latitude of second set of coords
 
-    returns
-    Series: distance between all pairs of lat/lon coords
+    Returns
+    -------
+    result : cudf.Series
+        The distance between all pairs of lat/lon coordinates
```

Comment:
Order seems to switch here?

```suggestion
        The distance between all pairs of lon/lat coordinates
```